### ISSUE TITLE
Sources for you to resolve some of your issues

### ISSUE THREAD

**Issue by jelegend** [DandelionSprout/NyxForWindows#1]
Hi, I saw your repor recently trying to run nyx on Windows. 

There is another repo on the same topic out there from 2 years ago 
(https://github.com/lunaken/nyx-for-windows)

The issue of graph,os error is solved there so I think you can look into that and fix those accordingly is possible

However the other repo's introduces two different bugs.
1. The colors are gone (show default white on running), other minor UI bugs
2. Menu options show gibberish

I am hoping that you could combine the best of both worlds so to speak and then publish it (maybe even contact the original nyx devs) so that wider community can benefit.

P.S. The one bug common in both is that because of Windows and lib issues it seems no one has yet to figure the alternative way to get computer usage stats working in absence of os.getuid() etc. If I figure out anything I'll add it here later on

**Comment by DandelionSprout** [comment]
I'll make sure to compare their `header.py` and `starter.py` files with mine, to see what changes they made to it and see if it's miraculously that easy to fix it. I can't give a timeframe promise, but at the time of writing, I aim to look into it within the next couple weeks.

**Comment by DandelionSprout** [comment]
Okay, so, a week late, I finally got around to looking into this tonight. His repo clued me on to how I should replace *all* instances of `os.uname` with `platform.uname`, including one instance in `tracker.py` that had previously slipped past me. Doing that (along with using `import platform` whenever needed), and it works. No errors, and the bandwidth automatically scrolls. I have made history.

Provided I can reproduce this once more but without all the heavy modifications I did during testing, a 3rd and final replacement file, this time for `tracker.py`, will be uploaded to this repo.
